annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -703,6 +703,15 @@ private TranscriptConsequenceSummary getCanonicalTranscript(VariantAnnotation gn
         }
     }
 
+    private String resolveDbSnpRs(VariantAnnotation gnResponse, MutationRecord mRecord) {
+        String dbSnpRs = null;
+        if (gnResponse.getColocatedVariants() != null && !gnResponse.getColocatedVariants().isEmpty()) {
+            dbSnpRs = gnResponse.getColocatedVariants().get(0).getDbSnpId();
```

Comment:
I guess the colocated variants are a collection/list? If the list contains rs0000001 and rs0000002, is it correct to return the first one? I would guess that these would be distinct variations which occur at the some location. So maybe rs0000001 is an A->C substitution and rs0000002 is an A->G substitution at the same genomic position. Would we need to look up the nucleotide variation involved in these two entries and only return the record that matches the query? (perhaps neither would match the query ... if the query was for an A->T substitution ... so then perhaps we should return nothing?)
I don't know the answer ... just raising the question. Is there any related information included in the output regarding colocated variants? There is a code called univar_id which would tell us the base substitution to see if we matched.

Reply:
@sheridancbio we assume the first one